src/main/java/eu/tornplayground/tornapi/models/faction/basic/member/OnlineStatus.java

```diff
@@ -0,0 +1,32 @@
+package eu.tornplayground.tornapi.models.faction.basic.member;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.io.Serializable;
+
+public enum OnlineStatus implements Serializable {
```

Comment:
Duplicate of `eu.tornplayground.tornapi.models.user.partial.LastActionStatus`, should probably become a shared class.

Reply:
moved some common classes into a package and fixed classes and tests